src/datasets/search.py

```diff
@@ -695,8 +695,10 @@ def search(self, index_name: str, query: Union[str, np.array], k: int = 10, **kw
                 The number of examples to retrieve.
 
         Returns:
-            - scores (`List[List[float]`): The retrieval scores of the retrieved examples.
-            - indices (`List[List[int]]`): The indices of the retrieved examples.
+            `(scores, indices)`:
+                A tuple of `(scores, indices)` where:
+                - **scores** (`List[List[float]`): the retrieval scores (`IndexFlatL2` score by default) of the retrieved examples
```

Comment:
it's `IndexFlatL2` by default for a faiss index, but not for an ElasticSearch index
```suggestion
                - **scores** (`List[List[float]`): the retrieval scores from either FAISS (`IndexFlatL2` by default) or ElasticSearch of the retrieved examples
```